### PR TITLE
chore: add code-owners

### PR DESCRIPTION
**Pull request by zmstone**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+apps/emqx_sasl/ @savonarola
+
+## Gateway
+apps/emqx_coap/ @HJianBo
+apps/emqx_exhook/ @HJianBo
+apps/emqx_exproto/ @HJianBo
+apps/emqx_lua_hook/ @HJianBo
+apps/emqx_lwm2m/ @HJianBo
+
+
```
Comment (HJianBo) [inline .github/CODEOWNERS:27]: nit: extra line?

**Review by thalesmg — DISMISSED** [review]

**Review by HJianBo — APPROVED** [review]
Awesome!!! This seems to be a great feature of GitHub

**Review by id — APPROVED** [review]

**Comment by id** [comment]
why is it happening?
```
===> Error generating relup: 
          No release upgrade script entry for gun-1.3.7 to gun-1.3.8 in file /__w/emqx/emqx/_build/emqx/rel/emqx/lib/gun-1.3.7/ebin/gun.appup
```

**Comment by zmstone** [comment]
> why is it happening?
> 
> ```
> ===> Error generating relup: 
>           No release upgrade script entry for gun-1.3.7 to gun-1.3.8 in file /__w/emqx/emqx/_build/emqx/rel/emqx/lib/gun-1.3.7/ebin/gun.appup
> ```

because the base was too old. now rebased.

**Review by HJianBo — APPROVED** [review]

**Review by JimMoen — APPROVED** [review]
awesome

**Review by id — APPROVED** [review]

**Pull request merged**